.github/workflows/pr_build_info.yml

```diff
@@ -0,0 +1,18 @@
+name: Pull Request CI Build Info
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  post_docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: mshick/add-pr-comment@v1
+      with:
+        message: |
+          **This PR will be available at** http://94.130.177.235/gha/PR${{ github.event.number }}/ **once building has completed.**
+          Every new commit will trigger a rebuild. Look at the "Checks" to see progress.
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        repo-token-user-login: 'Buildsystem' # The user.login for temporary GitHub tokens
+        allow-repeats: false # This is the default
```

Comment:
Add missing line ending, preferred on unix based systems

Reply:
Changed the setting in my IDE. Should have done it.